README.md

```diff
@@ -101,6 +101,8 @@ A new Docker image will be pushed from our [GitHub repository](https://github.co
 
 ## Development Troubleshooting and FAQ
 
+* **How to reset Metamask after restarting the chain** - You should first sign out of your account through the account icon on the top right. Then in the Metamask extension, click on the **Account** icon and go through **Settings --> Advanced**. Reset your transaction history through the **Reset Account** button. Afterwards, reconnect with the **Connect** button using Metamask. The client should be fully reset and ready to go.
```

Comment:
Good content. Could you put each sentence on a separate line? I don't think we have that rule in this repo, but we have tried to follow it for easier raw markdown readability.

Thanks!

Reply:
I'll get it done asap!